src/styles/blocks/_buy_button.scss

```diff
@@ -0,0 +1,21 @@
+.product__buy-button {
```

Comment:
buy-button is an element of product block, so it should be located in the file product.scss